frontend/app/controllers/Event.scala

```diff
@@ -167,6 +167,7 @@ trait Event extends Controller with ActivityTracking {
       trackAnon(EventActivity("eventThankYou", memberData, EventData(event), order.map(OrderData(_))))(request)
       event.service.wsMetrics.put("user-returned-to-thankyou-page", 1)
     }
+    event.service.wsMetrics.put("user-returned-to-thankyou-page-no-cookie", 1)
```

Comment:
You could also move the `trackAnon` code to here too. You can then limit queries on the dashboard by unique `orderId` in case of the user refresh problem.